charts/kyverno/Chart.yaml

```diff
@@ -33,8 +33,6 @@ annotations:
       url: https://kyverno.io/docs
   # valid kinds are: added, changed, deprecated, removed, fixed and security
   artifacthub.io/changes: |
```

Comment:
I think we can remove all these from the main branch.

Reply:
@realshuting Thank you for your feedback. should I remove all annotations from  ```artifacthub.io/changes: | ``` field.  or should I remove comments and artifacthub.io/links section which contains doc url?

Reply:
>should I remove all annotations from artifacthub.io/changes: |  field.

Yes.